contrib/lib/tests/templates.rs

```diff
@@ -116,5 +116,57 @@ mod templates_tests {
             let response = client.get("/tera/test").dispatch();
             assert_eq!(response.status(), Status::NotFound);
         }
+
+        #[cfg(debug_assertions)]
+        #[test]
+        fn test_template_reload() {
+            use rocket::local::Client;
+            use std::fs::File;
+            use std::io::Write;
+            use std::path::Path;
+            use std::thread;
+            use std::time::Duration;
+
+            fn write_file(path: &Path, text: &str) {
+                let mut file = File::create(path).expect("open file");
+                file.write_all(text.as_bytes()).expect("write file");
+                file.sync_all().expect("sync file");
+            }
+
+            const RELOAD_TEMPLATE: &'static str = "hbs/reload";
```

Comment:
`const`s should come before everything else. Note that we don't need the `'static` since that's implied.